libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -127,15 +127,10 @@ use crate::rest_api::sessions::AccessTokenIssuer;
 /// * `GET /biome/user/{id}` - Retrieve user with specified ID
 /// * `DELETE /biome/user/{id}` - Remove user with specified ID
 pub struct BiomeRestResourceManager {
-    // Disable lint warning, for now this is only used if the biome-credentials feature is enabled
```

Comment:
These should be changed to #[cfg(feature = "biome-rest-api")] until that feature is removed.